pages/common/gradle.md

```diff
@@ -3,19 +3,19 @@
 > Gradle is an open source build automation system.
 > More information: <https://gradle.org>.
 
-- Compile a package:
+- Assemble outputs and run checks:
 
 `gradle build`
 
-- Exclude test task:
+- Exclude the `test` task:
 
 `gradle build -x {{test}}`
 
-- Run in offline mode to prevent gradle from accessing the network during builds:
+- Run in offline mode to prevent Gradle from accessing the network during build:
 
 `gradle build --offline`
 
-- Clear the build directory:
+- Clear the output directory:
```

Comment:
What was wrong with `build`?

Reply:
Nothing wrong. I think I should revert this change. 😇 
https://docs.gradle.org/current/userguide/command_line_interface.html#cleaning_outputs

Reply:
If the docs call it `output` then it's fine of course.